docs/load-test-how-to-use.md

```diff
@@ -0,0 +1,115 @@
+    - If you see errors, there's a [troubleshooting guide](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/faq.md#troubleshooting) you can reference to resolve the issues.
+    - Check `ltagent.log` located in the `mattermost-load-test-ng` directory, and the server logs for details on errors, if any.
+    - We highly recommend becoming familiar with additional sections of documentation, including [using load-test agent API server](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/local_loadtest.md#running-a-load-test-through-the-load-test-agent-api-server) and [using load-test coordinator](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/local_loadtest.md#running-a-load-test-through-the-coordinator), since the Terraform deployment uses the latter method to execute the load-tests, and it'll be easier to debug the setup if the you understand these underlying concepts and principles.
+
+
+#### Testing changes in terraform
+
+Loadtest instances created with this framework achieve the same goals as mentioned above, only some of the things like creating a deployment, running a loadtest, etc. are automated.
+
+The steps to load-test a new feature in production, after testing new actions locally, includes the following:
+
+ - Review the [terraform_loadtest.md](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md) documentation.
+ - Some additional information on the above document
+    - When performing [this](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md#copy-and-modify-the-required-configuration) step, edit `MattermostLicenseFile` value to the path containing the license.
+        - The fields `MattermostDownloadURL` and `LoadTestDownloadURL` point to the latest `mattermost-server`, and `load-test package`, to be used in the load-test.
+        - That's the default option, when there's unmerged changes to `mattermost-server`, or `mattermost-load-test-ng`.
+            - Run `make build-linux` in the `mattermost-server` directory, change the `MattermostDownloadURL` value to the path containing the Mattermost executable. For example `file:///somepath/mattermost-server/bin/linux_amd64/mattermost`.
+            - Run `make package` in the `mattermost-load-test-ng` directory, change `LoadTestDownloadURL` value to the path containing the gzip of the load-test package. For example `file:///somepath/mattermost-load-test-ng/dist/v1.5.0-8-gd4f18cf/mattermost-load-test-ng-v1.5.0-8-gd4f18cf-linux-amd64.tar.gz`.
+    - Edit `SSHPublicKey` in `deployer.json` after setting up ssh.
+    - `go run ./cmd/ltctl deployment create`
+        - Limit operations of `deployment` to a single shell window.
```

Comment:
Why do we need to limit operations of `deployment` to a single window?

Reply:
Thanks for catching this, will remove this. This isn't true, since commands get stuck because of other reasons.